api/admin/controller.py

```diff
@@ -1834,6 +1836,13 @@ def collections(self):
                         ]
         protocols = self._get_integration_protocols(provider_apis, protocol_name_attr="NAME")
 
+        # If there are storage integrations, add a mirror integration
+        # setting to every protocol's 'settings' block.
+        mirror_integration_setting = self._mirror_integration_setting()
+        if mirror_integration_setting:
+            for protocol in protocols:
```

Comment:
Shouldn't this be more limited? It doesn't make sense for most of the protocols.

Reply:
I think it makes sense for every _collection_ protocol because of the cover image mirror thing.

Reply:
:+1: